### PR TITLE
T21654

### PR DESCRIPTION
**Pull request by GeorgesStavracas**
This PR introduces the actors at the password dialog asking whether the connection is metered or not. It also changes the wording of the title of the dialog, to make it less technical.

https://phabricator.endlessm.com/T21654

### CODE REVIEW

File: js/ui/components/networkAgent.js [hunk 1]
```diff
@@ -140,6 +145,104 @@ var NetworkSecretDialog = new Lang.Class({
...
+        // Title label
+        let title = new St.Label({ text: _("Is this connection paid or limited?"),
+                                   style_class: 'message-dialog-title',
+                                   x_expand: true,
+                                   x_align: Clutter.ActorAlign.START });
+        updatesLayout.attach(title, 1, 1, 1, 1);
+
+        // Subtitle label
+        let subtitle = new St.Label({ style_class: 'message-dialog-body',
+                                      text: _("Enable this option if this connection has usage restrictions, or is paid. This will disable automatic updates."),
```
Comment (nedrichards) [inline js/ui/components/networkAgent.js]: I really want to make sure we test this wording in market during the beta period.

**Comment by joaquimrocha** [comment]
Type in `This allows users to mark a connection as metered or not, and
tha[T] will`.

File: js/ui/components/networkAgent.js [hunk 2]
```diff
@@ -162,11 +267,52 @@ var NetworkSecretDialog = new Lang.Class({
...
+
+        // Store the Automatic Updates first
+        let userSetting = remoteConnection.get_setting(NM.SettingUser.$gtype);
+
+        if (!userSetting) {
+            userSetting = new NM.SettingUser();
+            remoteConnection.add_setting(userSetting);
+        }
+
+        let value = isMeteredConnection ? '0' : '1';
```
Comment (joaquimrocha) [inline js/ui/components/networkAgent.js]: You don't need to define a var for something that'll just be used inside the next line, right?

Comment (GeorgesStavracas) [reply js/ui/components/networkAgent.js]: Well, I just thought it'd make the following line shorter. No strong opinion here.

**Pull request merged**